src/Factory/ActivityPub/GroupFactory.php

```diff
@@ -48,7 +48,7 @@ public function create(Magazine $magazine): array
             ],
             'summary' => $magazine->description,
             'sensitive' => $magazine->isAdult,
-            'moderators' => $this->urlGenerator->generate(
```

Comment:
One thing I was wondering is if we should be setting both, to try to work with both lemmy and kbin in the future, as this seems like sort of an API break where we're saying we'll go with fep-1b12 rather than what kbin was planning?

Reply:
I'd say we wait until kbin implements something. Then we can look how we can support it